packages/zowe-explorer/src/utils/ProfilesUtils.ts

```diff
@@ -149,35 +153,34 @@ export class ProfilesUtils {
      */
-    public static async promptAndDisableCredentialManagement(): Promise<void> {
-        ZoweLogger.trace("ProfilesUtils.promptAndDisableCredentialManagement called.");
-        const noButton = vscode.l10n.t("No");
-        const yesGloballyButton = vscode.l10n.t("Yes, globally");
-        const yesWorkspaceButton = vscode.l10n.t("Only for this workspace");
-        const response = await Gui.warningMessage(
-            vscode.l10n.t("Zowe Explorer failed to activate since the default credential manager is not supported in your environment."),
-            {
-                items: [noButton, yesGloballyButton, yesWorkspaceButton],
-                vsCodeOpts: {
-                    modal: true,
-                    detail: vscode.l10n.t("Do you wish to disable credential management? (VS Code window reload will be triggered)"),
-                },
-            }
-        );
-        if (response === yesGloballyButton || response === yesWorkspaceButton) {
-            const scope = response === yesGloballyButton ? vscode.ConfigurationTarget.Global : vscode.ConfigurationTarget.Workspace;
-            await SettingsConfig.setDirectValue(Constants.SETTINGS_SECURE_CREDENTIALS_ENABLED, false, scope);
-            await vscode.commands.executeCommand("workbench.action.reloadWindow");
-        } else {
-            throw new imperative.ImperativeError({
-                msg: vscode.l10n.t(
-                    // eslint-disable-next-line max-len
-                    "Failed to load credential manager. This may be related to Zowe Explorer being unable to use the default credential manager in a browser based environment."
+    public static async disableCredentialManagement(): Promise<void> {
+        ZoweLogger.trace("ProfilesUtils.disableCredentialManagement called.");
+        const settingEnabled: boolean = SettingsConfig.getDirectValue(Constants.SETTINGS_SECURE_CREDENTIALS_ENABLED, true);
+        if (settingEnabled) {
+            this.PROFILE_SECURITY = false;
+            await SettingsConfig.setDirectValue(Constants.SETTINGS_SECURE_CREDENTIALS_ENABLED, false, vscode.ConfigurationTarget.Global);
+            await Gui.infoMessage(
+                vscode.l10n.t(
+                    "Zowe Explorer's default credential manager is not supported in your environment. Consider installing a custom solution for your platform. Click Reload to start Zowe Explorer without a credential manager."
                 ),
-            });
+                {
+                    items: [vscode.l10n.t("Reload window")],
+                }
+            );
+            await vscode.commands.executeCommand("workbench.action.reloadWindow");
+        }
+    }
+
+    public static checkDefaultCredentialManager(): boolean {
+        try {
+            ProfilesCache.requireKeyring();
+        } catch (_error) {
+            ZoweLogger.info(vscode.l10n.t("Default Zowe credentials manager not found on current platform."));
+            return false;
```

Comment:
Could we add the error message to the logger here in case we run into an issue with loading the credential manager? Otherwise, it could be difficult to track down why the default credential manager failed to load (could be useful when diagnosing credential manager bugs / customer issues).

Reply:
I don't want to log an error here as this will be the normal case for a container-based environment. I added more details about why this info message could have happened. What about that?

Reply:
Logging as `info` makes sense, thanks for adding that 👍